### PR TITLE
Well, big changes.

### PR DESCRIPTION
**Pull request by avaidyam**

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -49,25 +50,32 @@
...
                 </button>
-                <a class="navbar-brand" href="/">Purdue IEEE</a>
+<?php if($back_needed) : ?>
+                <a href="/"><button type="button" class="navbar-left btn btn-link btn-dark" style="height: 50px; padding: 0px 15px; font-size: 18px; line-height: 20px;" aria-label="Back">
+                    <i class="material-icons" style="color:white; font-size: 32px; margin-top: 4px;">arrow_back</i>
+                </button></a>
+<?php endif; ?>
+                <!--<img alt="Brand" src="...">-->
+                <div class="navbar-brand">
+                    <b><?php echo $title;?></b>
```
Comment (mattmolo) [inline header.php:54]: can we make this a link to `href="#"`. Should we have a back button on normal webpages? One of the common UX for website is top left corner goes to homepage.

Comment (avaidyam) [reply header.php:54]: @mattmolo Doesn't need to be a link to nothing... that's annoying to click and then go back to the same page.

Comment (mattmolo) [reply header.php:54]: It feels like a link it should be a link tho :P For copy paste purposes.

Still argue for the top left goes back to home, dik

Comment (avaidyam) [reply header.php:54]: Link to what!? It's pointless. Android Action Bars don't link to anything, MD says not to do that anyway.

File: js/bootstrap.js [hunk 2]
```diff
@@ -1,2363 +0,0 @@
...
-      if (!($(e.target).is('input[type="radio"]') || $(e.target).is('input[type="checkbox"]'))) e.preventDefault()
-    })
-    .on('focus.bs.button.data-api blur.bs.button.data-api', '[data-toggle^="button"]', function (e) {
-      $(e.target).closest('.btn').toggleClass('focus', /^focus(in)?$/.test(e.type))
-    })
-
-}(jQuery);
-
-/* ========================================================================
- * Bootstrap: carousel.js v3.3.5
```
Comment (mattmolo) [inline js/bootstrap.js:296]: da carousel

Comment (avaidyam) [reply js/bootstrap.js:296]: It was renamed and moved, genius.

File: header.php [hunk 3]
```diff
@@ -49,25 +44,32 @@
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a class="navbar-brand" href="/">Purdue IEEE</a>
+<?php if($back_needed) : ?>
+                <a href="/"><button type="button" class="navbar-left btn btn-link btn-dark" style="height: 50px; padding: 0px 15px; font-size: 18px; line-height: 20px;" aria-label="Back">
```
Comment (mattmolo) [inline header.php]: `float: left` to fix the mobile issue

File: header.php (hunk 1 above)
Comment (mattmolo) [reply header.php:54]: see android.com, facebook, twitter, gmail, hangouts website, google+ for top left home button

Comment (mattmolo) [reply header.php:54]: I like Google+ way, which doesn't have the page title be a link, but it has the top left button

**Comment by mattmolo** [comment]
TODO: Fix old learning.

File: committees/old_learning/index.php [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                    <a href="/rov/" class="list-group-item">Remotely Operated underwater Vehicle</a>
+                </div>
+            </div>
+
+            <!-- Content Column -->
+            <div class="col-md-7 text-dark col-md-offset-1">
+                <h2>Learning Committee</h2>
+                <p class="text-center"><img src="/learning/images/learning_header.png" alt="Learning Committee" style="width:90%"></p>
+                <p>The IEEE Learning Committee is dedicated to teaching any member the basics in Software, Hardware, and Mechanical concepts. These concepts vary from soldering, PCB design, JavaScript and Arduino Wiring programming, and 3D CAD modeling for 3D printing. The committee is structured in lessons that students participate in. Each lesson builds towards a final product that encompasses most if not all of the core skills that they will be learning.</p>
+                <p>Contact <a href="mailto:lee1509@purdue.edu">John Lee</a><br>If you’re interested in getting started at any time, let the committee head, John Lee, know via email.</p>
```
Comment (mattmolo) [inline committees/old_learning/index.php]: lets remove old contact info, and the next line after.

File: committees/old_learning/index.php [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+                    <a href="/mtts/" class="list-group-item">Microwave Theory &amp; Techniques Society</a>
+                    <a href="/racing/" class="list-group-item">Racing</a>
+                    <a href="/rov/" class="list-group-item">Remotely Operated underwater Vehicle</a>
+                </div>
+            </div>
+
+            <!-- Content Column -->
+            <div class="col-md-7 text-dark col-md-offset-1">
+                <h2>Learning Committee</h2>
+                <p class="text-center"><img src="/learning/images/learning_header.png" alt="Learning Committee" style="width:90%"></p>
```
Comment (mattmolo) [inline committees/old_learning/index.php]: yeah fix this image include

File: committees/old_learning/index.php [hunk 6]
```diff
@@ -0,0 +1,35 @@
+<?php
+    $title = 'IEEE Learning Committee';
+    include '../../header.php';
+?>
+
+    <!-- Content Row -->
+    <div class="well">
+        <div class="row">
+
+            <!-- Sidebar Column -->
```
Comment (mattmolo) [inline committees/old_learning/index.php]: da sidbar

File: committees/rov/index.php [hunk 7]
```diff
@@ -1,108 +1,3 @@
 <?php
-$title = 'IEEE ROV';
-include '../../header.php';
+    header("Location: ../rov/")
```
Comment (mattmolo) [inline committees/rov/index.php:3]: comment that this is for legacy support for temporary measures

File: committees/old_learning/index.php [hunk 8]
```diff
@@ -0,0 +1,35 @@
+<?php
+    $title = 'IEEE Learning Committee';
+    include '../../header.php';
+?>
+
```
Comment (mattmolo) [inline committees/old_learning/index.php:7]: comment this is about `legacy` support

File: header.php [hunk 9]
```diff
@@ -49,25 +44,32 @@
                     <span class="icon-bar"></span>
                     <span class="icon-bar"></span>
                 </button>
-                <a class="navbar-brand" href="/">Purdue IEEE</a>
+<?php if($back_needed) : ?>
```
Comment (mattmolo) [inline header.php]: this indenting is atrocious though..

File: index.php [hunk 10]
```diff
@@ -111,10 +106,7 @@
             <div class="col-md-12">
                 <div class="media">
                     <div class="pull-left">
-                        <span class="fa-stack fa-2x">
-                            <i class="fa fa-circle fa-stack-2x text-primary"></i>
-                            <i class="fa fa-globe fa-stack-1x fa-inverse"></i>
-                        </span>
+                            <i class="material-icons" style="color:#072a40; font-size: 32px; margin-left:16px;">insert_emoticon</i>
```
Comment (mattmolo) [inline index.php:109]: how about ieee logo?

File: index.php [hunk 11]
```diff
@@ -260,14 +228,11 @@
...
-                    <span class="fa-stack fa-5x">
-                        <i class="fa fa-circle fa-stack-2x text-primary"></i>
-                        <i class="fa fa-envelope-o fa-stack-1x fa-inverse"></i>
-                    </span>
+                    <i class="material-icons" style="color:#072a40; font-size: 96px;">email</i>
                 </div>
                 <div class="panel-body">
                     <h4>Contact Us</h4>
-                    <p>Contact us for more information at<br><a href="mailto:IEEE@purdue.edu">IEEE@purdue.edu</a>, or stop by at EE 014</p>
+                    <p>Contact us for more information at<br><a href="mailto:IEEE-Info@purdueieee.org">IEEE-Info@purdueieee.org</a>, or stop by at EE 014</p>
```
Comment (mattmolo) [inline index.php]: this is wrong.. I don't think things from master got merged correctly. Regresses #51

File: index.php (hunk 10 above)
Comment (avaidyam) [reply index.php:109]: I was gonna do that! 😄 It's an improvement for later though.

File: index.php (hunk 11 above)
Comment (avaidyam) [reply index.php]: Yeah it didn't get merged right. 👎

File: rov/sponsors/2015.php [hunk 12]
```diff
@@ -91,37 +69,37 @@
...
                 </div>
 
                 <div class="row" style="margin-top: 20px">
                     <div class="col-md-4 text-center">
-                        <a href="http://wabashpower.com" target="_blank"><img src="/committees/rov/images/WabashPower.jpg" alt="Wabash Power Equipment" style="width:100%"></a>
+                        <a href="http://wabashpower.com" target="_blank"><img src="/rov/images/WabashPower.jpg" alt="Wabash Power Equipment" style="width:100%"></a>
                     </div>
                     <div class="col-md-4 text-center">
-                        <a href="http://www.binder-world.com/us/Company" target="_blank"><img src="/committees/rov/images/Binder.png" alt="Binder" style="width:100%"></a>
+                        <a href="http://www.binder-world.c../Company" target="_blank"><img src="/rov/images/Binder.png" alt="Binder" style="width:100%"></a>
```
Comment (mattmolo) [inline rov/sponsors/2015.php]: this looks wrong...

Comment (avaidyam) [reply rov/sponsors/2015.php]: I don't even know how that screwed up -- oh wait I do. I suck at sed. 😢

File: js/bootstrap.js [hunk 13]
```diff
@@ -1,2363 +0,0 @@
-/*!
```
Comment (mattmolo) [inline js/bootstrap.js:1]: Oh I see, this is the unminified version, which is useful for debugging in broken JS. It's somewhat useful.. it's not included anywhere, so I don't think there's harm in keeping it when we break the carousel.

Comment (avaidyam) [reply js/bootstrap.js:1]: Honestly, I think it's better to remove it for now. Again, this is a short term UI fix for a long term overhaul in MDL.

**Review by mattmolo — APPROVED** [review]

**Pull request merged**